File: Web/ISnack/src/main/java/me/codeminions/bean/db/Snack.java
package me.codeminions.bean.db;

import com.google.gson.annotations.Expose;

public class Snack {       //对应表t_snack

    @Expose
    private int snackID;
    @Expose
    private String name;
    @Expose
    private String origin;
    @Expose
    private String shelf;
    @Expose
    private float mark;
    @Expose
    private String img;   // 图片的url

    public Snack(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getOrigin() {
        return origin;
    }

    public void setOrigin(String origin) {
        this.origin = origin;
    }

    public String getShelf() {
        return shelf;
    }

    public void setShelf(String shelf) {
        this.shelf = shelf;
    }

    public float getMark() {
        return mark;
    }

    public void setMark(float mark) {
        this.mark = mark;
    }

    public String getImg() {
        return img;
    }

    public void setImg(String img) {
        this.img = img;
    }

    public int getSnackID() {
        return snackID;
    }

    public void setSnackID(int snackID) {
        this.snackID = snackID;
    }
}

File: App/ISnack/factory/src/main/java/me/codeminions/factory/data/bean/Snack.kt
package me.codeminions.factory.data.bean

import java.io.Serializable

data class Snack(var snackID: Int? = null,
                 var name: String? = null,
                 var origin: String? = null,
                 var shelf: String? = null,
                 var mark: Float = 0.toFloat(),
// 图片的URL
                 var img: String? = null) : Serializable {

    constructor(name: String, mark: Float) : this() {
        this.name = name
        this.mark = mark
    }

    constructor(id: Int, name: String, mark: Float) : this() {
        this.snackID = id
        this.name = name
        this.mark = mark
    }
}